db/migrate/20230525073654_create_deals.rb

```diff
@@ -0,0 +1,17 @@
+class CreateDeals < ActiveRecord::Migration[7.0]
+  def change
+    create_table :deals do |t|
+      t.string :title
+      t.string :description
+      t.integer :price_in_cents
+      t.integer :discount_price_in_cents
+      t.integer :quantity, default: 0
+      t.datetime :publish_date
+      t.datetime :published_date
```

Comment:
let's rename these to published_at and publish_at if these are date time and not dates

Reply:
done